programs/launcher/main.cpp

```diff
@@ -1092,6 +1098,10 @@ launcher_def::start_all (string &gts, launch_modes mode) {
         for (auto &inst : h.instances) {
           try {
             launch (inst, gts);
+          } catch (fc::exception& fce) {
+             cerr << "unable to launch " << inst.name << "fc::exception=" << fce.to_detail_string() << endl;
+          } catch (std::exception& stde) {
+             cerr << "unable to launch " << inst.name << "std::exception=" << stde.what() << endl;
           } catch (...) {
```

Comment:
Can we get a space added. `" fc::exception="` and `" std::exception="`. Both here and above.